pom.xml

```diff
@@ -1612,7 +1614,7 @@
       </properties>
       <activation>
         <property>
-          <name>spark3</name>
+          <name>spark3.1.x</name>
```

Comment:
fix the name conflict with spark3 profile, or it will use spark3.1.2 when running `mvn clean package -Dscala-2.12 -Dspark3 -DskipTests`